apps/omg_performance/lib/performance.ex

```diff
@@ -90,7 +90,7 @@ defmodule OMG.Performance do
   ```
   %{
     destdir: ".", # directory where the results will be put
-    geth: "http://localhost:8545",
+    geth: System.get_env("ETHEREUM_RPC_URL"),
```

Comment:
and this one is the default value - I think you should be able to use `ETHEREUM_RPC_URL` while invoking the perftest and keep the `localhost:8545` default here.